README.md

```diff
@@ -900,6 +900,79 @@ _You can enable the following settings in Xcode by running [this script](resourc
     …
   }
   ```
+  
+  </details>
+
+* <a id='wrap-multiline-conditional-assignment'></a>(<a href='#wrap-multiline-conditional-assignment'>link</a>) **Add a line break after the assignment operator (`=`) before a multi-line `if` or `switch` expression**, and indent the following `if` / `switch` expression. If the declaration fits on a single line, a line break not required. [![SwiftFormat: wrapMultilineConditionalAssignment](https://img.shields.io/badge/SwiftFormat-wrapMultilineConditionalAssignment-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapMultilineConditionalAssignment)
+
+  <details>
+
+  #### Why?
+  
+  This makes it so that `if` and `switch` expressions always have the same "shape" as standard `if` and `switch` statements, where:
+  1. The `if` / `switch` keyword is always the left-most token on a dedicated line of code
+  2. The conditional branches are always to the right of and below the `if` / `switch` keyword
+
+  This is most consistent with the formatting of `if` / `switch` statements in other contexts, and thus makes it easier to recognize that the code is using an `if` or `switch` expression at a glance. 
+  
+  ```swift
+  // WRONG. Should have a line break after the `=`. 
+  let planetLocation = if let star = planet.star {
+    "The \(star.name) system"
+   } else {
+    "Rogue planet"
+  }
+
+  // WRONG. The `=` should be on the line before the `if` / `switch` expression. 
+  let planetLocation 
+    = if let star = planet.star {
+      "The \(star.name) system"
+    } else {
+      "Rogue planet"
+    }
+
+  // WRONG. `if` expression should be indented.
+  let planetLocation =
+  if let star = planet.star {
+    "The \(star.name) system"
+  } else {
+    "Rogue planet"
+  }
+    
+  // RIGHT 
+  let planetLocation = 
+    if let star = planet.star {
+      "The \(star.name) system"
+    } else {
+      "Rogue planet"
+    }
+    
+  // RIGHT 
+  let planetLocation = 
+    if let star = planet.star {
+      "The \(star.name) system"
+    } else {
+      "Rogue planet"
+    }
+    
+  // RIGHT
+  let planetType: PlanetType =
+    switch planet {
+    case .mercury, .venus, .earth, .mars:
+      .terrestrial
+    case .jupiter, .saturn, .uranus, .neptune:
+      .gasGiant
+    }
+    
+  // ALSO RIGHT. A line break is not required because the declaration fits on a single line. 
+  let planet = if useFirst { mercury } else { neptune }
+
+  // ALSO RIGHT. A line break is permitted if it helps with readability.
+  let planet =
+    if useFirst { mercury } else { neptune }
```

Comment:
Would this be valid?

```swift
let moonName = if let moon = moon { moon.name } else { "none" }
```

It's a bit odd to see two `=` on the same line... I think it's too much of a nuance to create a rule around it, but I wonder if these last two examples would be more powerful if we covered the double `=` situation instead.

```suggestion
  // ALSO RIGHT. A line break is not required because the declaration fits on a single line.
  let moonString = if let moon = moon { moon.name } else { "none" }

  // ALSO RIGHT. A line break is permitted if it helps with readability.
  let moonString =
    if let moon = moon { moon.name } else { "none" }
```

The double `=` situation seems like a great time to add a line break even if it's not strictly required.

Reply:
> Would this be valid?
>
> `let moonName = if let moon = moon { moon.name } else { "none" }`

Yeah, that example is perfectly valid. I like the idea of using an `=` in the example. I'll just change it to something other than `if let moon = moon {`, which our style guide would collapse to `if let moon {`.